src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/ProcessAppSI.cs

```diff
@@ -275,6 +308,37 @@ public async Task DispatchProcessEventsToStorage(Instance instance, List<Instanc
             return events;
         }
 
+        private InstanceEvent CopyInstanceEventValue(InstanceEvent e)
+        {
+            return new InstanceEvent
+            {
+                Created = e.Created,
+                DataId = e.DataId,
+                EventType = e.EventType,
+                Id = e.Id,
+                InstanceId = e.InstanceId,
+                InstanceOwnerPartyId = e.InstanceOwnerPartyId,
+                ProcessInfo = new ProcessState
+                {
+                    Started = e.ProcessInfo?.Started,
+                    CurrentTask = new ProcessElementInfo
+                    {
+                        Flow = e.ProcessInfo?.CurrentTask.Flow,
+                        AltinnTaskType = e.ProcessInfo?.CurrentTask.AltinnTaskType,
+                        ElementId = e.ProcessInfo?.CurrentTask.ElementId,
+                        Name = e.ProcessInfo?.CurrentTask.Name,
+                        Started = e.ProcessInfo?.CurrentTask.Started,
+                        Ended = e.ProcessInfo?.CurrentTask.Ended,
+                        Validated = e.ProcessInfo?.CurrentTask.Validated
+                    },
+
+                    StartEvent = e.ProcessInfo?.StartEvent
+                },
+                User = e.User
```

Comment:
Here we copy a reference instead of making a new PlatformUser. Intentional?

Reply:
As the user won't change during the call using the reference does not create an issue.

Reply:
copying value instead